addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -40,6 +40,7 @@ class Im_LivechatReportChannel(models.Model):
     partner_id = fields.Many2one('res.partner', 'Operator', readonly=True)
     handled_by_bot = fields.Integer("Handled by Bot", readonly=True, aggregator="sum")
     handled_by_agent = fields.Integer("Handled by Agent", readonly=True, aggregator="sum")
+    visitor_partner_id = fields.Many2one('res.partner', string="Visitor Partner", readonly=True)
```

Comment:
mix single/double quotes. we've been using double quotes for a while in discuss/livechat modules.